test/functional/feature_taproot.py

```diff
@@ -517,9 +517,9 @@ def random_checksig_style(pubkey):
     """Creates a random CHECKSIG* tapscript that would succeed with only the valid signature on witness stack."""
     return bytes(CScript([pubkey, OP_CHECKSIG]))
```

Comment:
I think this function should be refactored since the code after that line won't be reachable since there is a return right at the beginning of the function.

Reply:
Mind creating a pull for that, since it is completely unrelated to the changes here? The correct fix might be to remove this line

Reply:
I did it. See [#21081](https://github.com/bitcoin/bitcoin/pull/21081)